js/binance.js

```diff
@@ -4147,21 +4147,29 @@ module.exports = class binance extends Exchange {
             'asset': currency['id'],
             'amount': this.currencyToPrecision (code, amount),
         };
-        request['type'] = this.safeString (params, 'type');
+        const type = this.safeString (params, 'type');
+        if (type !== undefined) {
+            request['type'] = type;
+            params = this.omit (params, 'type');
+        }
```

Comment:
Why do we need to add these type-related lines at all? I mean, the extend-call does exactly the same thing.

```suggestion
```